test/user.spec.js

```diff
@@ -0,0 +1,63 @@
+const request = require('supertest');
+const app = require('.././app');
+
+describe('POST / signup users', () => {
+  const user = {
+    email: 'jose@wolox.com.ar',
+    name: 'jose',
+    lastName: 'perez',
+    password: 'asdasdasd654'
+  };
+
+  it('should succeed returning the created user', () => {
+    request(app)
+      .post('/users')
+      .send(user)
+      .then(res => {
+        expect(res.status).toBe(201);
+        expect(res.body.email).toBe(user.email);
+      });
+  });
+
+  test('should fail for the existence of the email', () => {
+    request(app)
+      .post('/users')
+      .send(user)
+      .then(res => {
+        expect(res.status).toBe(201);
+        expect(res.body.email).toBe(user.email);
+      });
+    request(app)
```

Comment:
This should be after first request, you have two parallel executions here, also you can use model directly for create user instead of call several times to same endpoint